src/kudu/consensus/raft_consensus.h

```diff
@@ -200,6 +200,12 @@ class RaftConsensus : public std::enable_shared_from_this<RaftConsensus>,
   // If the failure detector is already disabled, has no effect.
   void DisableFailureDetector();
 
+  // Pauses outgoing votes from this server during elections, if set to true.
+  void SetWitholdVotes(bool withold_votes);
+
+  // Rejects AppendEntries RPCs, if set to true.
+  void SetRejectAppendEntries(bool reject_append_entries);
```

Comment:
nit: SetRejectUpdateConsensusForTests() would be more in line with the implementation and the gflag